Lecture 3/GithubApp/app/src/main/java/mff/mdp/demoapp/ui/UserCard.kt

```diff
@@ -0,0 +1,96 @@
+package mff.mdp.demoapp.ui
+
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight.Companion.Bold
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import mff.mdp.demoapp.R
+
+@Composable
+fun UserCard(
+    name: String,
+    url: String,
+    imageUrl: String,
+    modifier: Modifier = Modifier
+) {
```

Comment:
This is maybe just matter of style, but I would provide also Composable `fun UserCard(user: User, modifier: Modifier = Modifier)` which will call this function

Reply:
Imho I prefer it this way - it is a bit more reusable, and we are not passing anything that we are not using in the composable.